src/main/java/com/m4thg33k/tombmanygraves/invman/InventoryHolder.java

```diff
@@ -62,7 +62,7 @@ public void grabPlayerData(EntityPlayer player) {
         playerName = player.getName();
         compound.setString(PLAYER_NAME, playerName);
 
-        setTimestamp(new SimpleDateFormat("MM_dd_YYYY_HH_mm_ss").format(new Date()));
+        setTimestamp(new SimpleDateFormat("YYYY-MM-dd'T'HH:mm:ss'Z'").format(new Date()));
```

Comment:
The "Z" you print out after the time says that the timestamp is in "Zulu", i.e. UTC. However, format() produces a timestamp in local time, not in UTC.

You want "yyyy-MM-dd'T'HH:mm:ss.SSSXXX" (or "yyyy-MM-dd'T'HH:mm:ssXXX") as given in the JavaDoc of SimpleDateFormat as example for ISO 8601.